# Bioinformatic Tool Compendium (BTC)

## Databases
- [National Center for Biotechnology Information (NCBI)](https://www.ncbi.nlm.nih.gov/) | Fundamental database containing biomedical and genomic information & tools
   - [PubMed](https://pubmed.ncbi.nlm.nih.gov/) | Literature database focused on biomedicine and health searches abstracts of articles with full text links
   - [PubMed Central (PMC)](https://www.ncbi.nlm.nih.gov/pmc/) | Component of PubMed contains free full text articles

- [UniProt](https://www.uniprot.org/) | Fundamental protein database

- [KEGG Database](https://www.genome.jp/kegg/) | Fundamental biological database with entry points into metabolic pathway maps, genes, proteins, biochemical reactions, and more.

- [BRENDA Enzyme Database](https://www.brenda-enzymes.org/) | Fundamental database on enzymes and related information

- [RCSB Protein Data Bank (PDB)](https://www.rcsb.org/) | Fundamental protein database with 3D structural information

- [ExPASy PROSITE](https://prosite.expasy.org/) | Database of protein domains, families, and functional sites.

- [Genomic tRNA Database](http://lowelab.ucsc.edu/GtRNAdb/) | Automated annotated tRNA gene predictions using tRNAscan-SE

- [The European Bioinformatics Institute (EMBL-EBI)](https://www.ebi.ac.uk/) | Fundamental database with bioinformatic databases and tools

- [RegulonDB Database](https://regulondb.ccg.unam.mx/ ) | Curated database on transcriptional regulation in *Escherichia coli* K-12

- [Bionumbers](https://bionumbers.hms.harvard.edu/search.aspx) | Describes useful number values in biology

- [BioCyc](https://biocyc.org) | Collection of pathway & genomic databases for model organisms
   - [EcoCyc](https://ecocyc.org/) | Curated database on genome, transcriptional regulation transporters & metabolic pathways in *Escherichia coli* K-12 MG1655

- [Regprecise](https://regprecise.lbl.gov/index.jsp ) | Contains curated and propagated predictions for regulons in prokaryotic genomes

- [Aging Circadian Database](https://circaage.shinyapps.io/circaage/) | Contains circadian gene expression data data from various ages and tissues

- [Database Commons](https://ngdc.cncb.ac.cn/databasecommons/) | Curated catalog of biological databases

- [circBase](http://www.circbase.org/) | Circular RNA database

- [SGD](https://www.yeastgenome.org/) | Comprehensive database on *Saccharomyces cerevisiae*

- [TAIR](https://www.arabidopsis.org/) | Contains data relevant to *Arabidopsis thaliana*

- [Atlas of Macromolecules](https://www.bioinformatics.org/molvis/atlas/atlas.htm) | Database containing data on about 160 macromolecules

- [circBank](http://www.circbank.cn/index.html ) | Circular RNA database

## Webtool Platforms
Suites of bioinformatic tools

- [CoGe](https://genomevolution.org/coge/) 

- [MPI Bioinformatics Toolkit](https://toolkit.tuebingen.mpg.de/) 

- [Softberry](http://www.softberry.com/)  

- [Elixir Bio.tools](https://bio.tools/t)

- [Sequence Manipulation Suite (SMS)](http://www.bioinformatics.org/sms2/index.html)

- [SeWeR](https://www.bioinformatics.org/sewer/)

- [Prabi](https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_server.html)

## Sequence Alignment Tools
Positions various sequences with similar portions in line with one another to observe possible homology

### Two Sequence Alignment Tools
- [Pairwise Align DNA](http://www.bioinformatics.org/sms2/pairwise_align_dna.html)

- [Pairwise Align Protein](http://www.bioinformatics.org/sms2/pairwise_align_protein.html)

- [PIR Pairwise Alignment](https://proteininformationresource.org/pirwww/search/pairwise.shtml)

- [LALIGN](https://www.ebi.ac.uk/Tools/psa/lalign/)

### Multiple Sequence Alignment Tools
- [Clustal Omega](https://www.ebi.ac.uk/Tools/msa/clustalo/ ) 

- [MultAlin](http://multalin.toulouse.inra.fr/multalin/cgi-bin/multalin.pl ) 

- [T-Coffee](http://tcoffee.crg.cat/apps/tcoffee/result?rid=24cdc9c7 ) 

- [EMBL-EBI MSA Tools](https://www.ebi.ac.uk/Tools/msa/)

- [MSAReveal](https://www.bioinformatics.org/msareveal/index2.html)

## Dotplot Generators
Produces dot plots for comparing sequences post alignment

- [Dotlet JS](https://dotlet.vital-it.ch/ ) 

- [Dotlet](https://myhits.sib.swiss/cgi-bin/dotlet ) 

- [Dotmatcher](https://www.bioinformatics.nl/cgi-bin/emboss/dotmatcher)

### Genome Comparison
- [CoGe: SynMap](https://genomevolution.org/CoGe/SynMap.pl )

- [CoGe: SynMap3D](https://genomevolution.org/coge/SynMap3D.pl) 

## Sequence Scanning
Specialized tools for scanning sequences against knowledgebase and specific functional areas of a sequence or genome 

- [BLAST](https://blast.ncbi.nlm.nih.gov/Blast.cgi) | The quintessential tool for identifying sequences

- [NCBI: ORFfinder](https://www.ncbi.nlm.nih.gov/orffinder/ ) | Searches for open reading frames in a sequence 

- [tRNAscan-SE ](http://lowelab.ucsc.edu/tRNAscan-SE/ ) | Uses predictive software to scan for possible tRNAs

- [GeneMark gene prediction](http://exon.gatech.edu/GeneMark/) | Implements unsupervised training to identify protein coding genes

- [FGENESH](http://www.softberry.com/berry.phtml?topic=fgenesh&group=programs&subgroup=gfind) | Gene structure prediction tool

- [Ribovore](https://github.com/ncbi/ribovore) | A collection of tools for subunit RNA detection

- [PeCoP](https://www.bioinformatics.org/pecop/) | A PSI-BLAST tool for finding persistently conserved positions

- [TargetScanHuman 8.0](https://www.targetscan.org/vert_80/ ) | Searches for predicted micro RNA targets in mammals

## Sequence Visualization & Editing
- [WebLogo About](https://weblogo.berkeley.edu/ ) | Generates sequence logos

- [Benchling](https://benchling.com/editor ) | Platform for conducting biotechnology research allowing for easy gene annotation, visualization, and more.

- [Savvy Plasmid Map](https://www.bioinformatics.org/savvy/)

- [NUPACK](https://old.nupack.org/ ) | Webserver for analyzing nucleic acid structures

- [Sequence Massager](https://biomodel.uah.es/en/lab/cybertory/analysis/massager.htm) | Allows for easy sequence modification including automatic transcription

- [Translate](https://web.expasy.org/translate/) | Easy tool for converting DNA/RNA sequences into amino acids

## Motifs & Protein Domains Identification
- [ScanProsite](https://prosite.expasy.org/scanprosite/) | Scans sequences against collection of motifs

- [InterPro](http://www.ebi.ac.uk/interpro/search/sequence/) | Classifies proteins into families using predictive algorithm and has curated databases

- [NCBI Conserved Domain Search](https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi ) | Identifies conserved domains within a sequence

- [Softberry Tools](http://www.softberry.com/berry.phtml?topic=index&group=programs&subgroup=promoter) | Suite of tools for scanning for motifs and domains

- [Introduction MEME Suite](https://meme-suite.org/meme/index.html ) | A collection of motif-based analysis tools

Relevant Databases:
- [SCOP| Structural Classification of Proteins](https://scop.mrc-lmb.cam.ac.uk/ ) | Contains detailed information concerning protein folds and related structures

- [CATH: Protein Structure Classification Database at UCL](http://www.cathdb.info/ ) | Database of protein structures

- [Conserved Domains Database (CDD)](https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml ) | Tools for analyzing conserved domains and protein classifications

- [COXPRESdb](https://coxpresdb.jp/) | Database with information concerning co-regulated gene relationships


## Structure Visualization
Group of tools for producing 2D & 3D structural data

- [Swiss-Model](https://swissmodel.expasy.org/interactive)

- [Protter interactive protein feature visualization](https://wlab.ethz.ch/protter/start/) | 2D protein visualizing tool of polypeptide sequence and predicted cellular location

- [iCn3D: Web-based 3D Structure Viewer](https://www.ncbi.nlm.nih.gov/Structure/icn3d/full.html) | Loads PDB files and generates 3D structures 

- [Cn3D](https://www.ncbi.nlm.nih.gov/Structure/CN3D/cn3d.shtml)

- [TM-align](https://zhanggroup.org//TM-align/) | Algorithm for superpositioning two protein structures using residue-residue alignment

- [US-align](https://zhanggroup.org/US-align/) | Extended from TM-align and compares 3D structures (proteins, RNA, & DNA) 

- [I-TASSER](https://zhanggroup.org/I-TASSER/)

- [RNAstructure](https://rna.urmc.rochester.edu/RNAstructureWeb/) | Contains tools for RNA secondary structure prediction

- [Phyre2](http://www.sbg.bio.ic.ac.uk/~phyre2/html/page.cgi?id=index)

- [Rasmol](http://www.openrasmol.org/)

- [Swiss-PDB Viewer](https://spdbv.unil.ch/)

- [GRASP](http://honig.c2b2.columbia.edu/grasp)

- [UCSF Chimera](http://www.cgl.ucsf.edu/chimera/)

- [Jmol](https://jmol.sourceforge.net/)

- [Jpred4](http://www.compbio.dundee.ac.uk/jpred4/index.html)

- [Protein Secondary Structure Tools](https://molbiol-tools.ca/Protein_secondary_structure.htm) | Group of programs for predicting protein secondary structure

Relevant Databases:
- [Alphafold](https://alphafold.ebi.ac.uk/)

- [Swiss-Model](https://swissmodel.expasy.org/)

- [RCSB PDB](https://www.rcsb.org/)


## Primer Development
- [Primer3Plus](https://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi) | Uses Primer3 software to produces primers to detect a given sequence

- [Primer3](https://bioinfo.ut.ee/primer3/) | Original software for generating primers for sequences

- [PrimerX](https://www.bioinformatics.org/primerx/)


## Restriction Enzymatic Tools
- [NEB List of Recognition Specificities](https://www.neb.com/tools-and-resources/selection-charts/alphabetized-list-of-recognition-specificities) | Endonuclease recognition sequences and associated enzyme

- [NEBcutter V2.0](http://nc2.neb.com/NEBcutter2/ ) | Identify cut sites for a DNA sequence

## Genome Browsers
- [Seed Viewer](https://pubseed.theseed.org/?page=OrganismSelect) | Curated genomic data for comparative genomics

- [JGI GOLD](https://gold.jgi.doe.gov/) | Genomic database

- [IslandViewer 4](https://www.pathogenomics.sfu.ca/islandviewer/)

- [UCSC Genome Browser](http://genome.ucsc.edu/cgi-bin/hgGateway)

- [Ensembl](http://useast.ensembl.org/index.html)

- [Ensembl Bacteria](http://bacteria.ensembl.org/index.html)

- [UCSC Archaeal Genome Browser](http://archaea.ucsc.edu/)

- [Pseudomonas Genome DB](https://www.pseudomonas.com/)

- [MicroScope: MAGE](https://mage.genoscope.cns.fr/microscope/home/index.php ) | Microbial Genome Annotation & Analysis Platform


## Phylogeny Tools
- [NGPhylogeny.fr](https://ngphylogeny.fr/) | Generates phylogenetic trees
    - [Phylogeny.fr](http://phylogeny.lirmm.fr/phylo_cgi/index.cgi ) |Previous version

- [Simple Phylogeny](https://www.ebi.ac.uk/Tools/phylogeny/simple_phylogeny/)

- [SeaView](https://doua.prabi.fr/software/seaview)

- [MEGA 11](https://www.megasoftware.net/)

## Protein Analysis Tools
- [Phobius](https://phobius.sbc.su.se/ ) 

- [SignalP 5.0 Services DTU Health Tech](https://services.healthtech.dtu.dk/service.php?SignalP-5.0 ) 

- [TMHMM 2.0 Services DTU Health Tech](https://services.healthtech.dtu.dk/service.php?TMHMM-2.0 ) 

- [TargetP 2.0 Services DTU Health Tech](https://services.healthtech.dtu.dk/service.php?TargetP-2.0 ) 

- [ExPASy Compute pI/Mw tool](https://web.expasy.org/compute_pi/ ) 

- [Protein Localization | MBInfo](https://www.mechanobio.info/the-cell/protein-localization/ ) 

- [Immunomedicine Group: Tools &gt;&gt; SIAS](http://imed.med.ucm.es/Tools/sias.html ) 

- [MitoMiner 4.0: Home](https://mitominer.mrc-mbu.cam.ac.uk/release-4.0/begin.do ) 

- [PRATT](https://web.expasy.org/pratt/ ) 


## Metagenomic Tools
Tools and packages for conducting metagenomic analysis

- [MicrobiomeAnalyst](https://www.microbiomeanalyst.ca/ ) | Statistical software for analyzing metagenomic data

- [DADA2](https://benjjneb.github.io/dada2/)

- [Mothur](https://mothur.org/)

- [Qiime2](https://qiime2.org/)

## Conversion Software
Useful tools for converting different between different filetypes

- [RNAComposer](https://rnacomposer.cs.put.poznan.pl/tools;jsessionid=945AA52BC80AD4C9A009D898D83F51CB ) | Converts RNA structure files between different formats

- [EMBOSS Seqret](https://www.ebi.ac.uk/Tools/sfc/emboss_seqret/)



## Forums
Helpful Forums for bioinformatics discussion

- [Biostars](https://www.biostars.org/t/Forum/)

- [SEQanswers](https://www.seqanswers.com/forum/bioinformatics/bioinformatics-aa)

- [Bioinformatics.org](http://www.bioinformatics.org/forum/forum.php?forum_id=1300&et=0)

- [Stack Overflow](https://stackoverflow.com/)

- [OmicsForum](https://omicsforum.ca/ ) | Community for various analysis tools related to metabolomics, metagenomics, proteomics, and more

- [QIIME2 Forum](https://forum.qiime2.org/)


## Old/Decommissioned Databases & Tools

- [MicrobesOnline](http://www.microbesonline.org/ ) | Database centered on microbial data CAUTION Last updated is June 2009

- [Xfam](http://xfam.org/ ) | Decommissioned collection of databases for families of Proteins, DNA, and RNA CAUTION Inactive

- [RNAmmer 1.2 Server](http://www.cbs.dtu.dk/services/RNAmmer/) | Identified related sequences to ribosomal subunits

- [GeneBee Motifs](http://www.genebee.msu.su/services/dhm/advanced.html) |  
