<h1 id="bioinformatic-tool-compendium-btc-">Bioinformatic Tool Compendium (BTC)</h1>

$\color{#3BBB27}\textsf{\Large\kern{0.2cm}\normalsize [web]}$ = web-based &nbsp;&nbsp;&nbsp;&nbsp; $\color{#FF0000}\textsf{\Large\kern{0.2cm}\normalsize [app]}$ = local install &nbsp;&nbsp;&nbsp;&nbsp; $\color{#D29922}\textsf{\Large\kern{0.2cm}\normalsize [cli]}$ = command-line

<h2 id="databases">Databases</h2>
<p>Fundamentals</p><ul>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information (NCBI)</a></b> | $\color{#3BBB27}\textsf{\Large\kern{0.2cm}\normalsize [web]}$  | Centralized repository of biomedical & genetic data<be></summary><code><a href="https://www.ncbi.nlm.nih.gov/guide/all/">[docs]</a></code></details>
<details><summary><b><a href="https://pubmed.ncbi.nlm.nih.gov/">PubMed</b></a> | Literature database focused on biomedicine and health searches abstracts of articles with full text links<br></summary><code></code></details></b>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/pmc/">PubMed Central (PMC)</b></a> | Component of PubMed contains free full text articles</summary><code></code></details>
<details><summary><b><a href="https://www.uniprot.org/">UniProt</b></a> | Database on proteins and their associated functions<br></summary><code></code></details>
<details><summary><b><a href="https://www.genome.jp/kegg/">KEGG Database</b></a> | Biological database with entry points into metabolic pathway maps, genes, proteins, biochemical reactions, and more.<br></summary><code></code></details>
<details><summary><b><a href="https://www.brenda-enzymes.org/">BRENDA Enzyme Database</b></a> | Database on enzymes and related information<br></summary><code></code></details>
<details><summary><b><a href="https://prosite.expasy.org/">ExPASy PROSITE</b></a> | Database of protein domains, families, and functional sites.<br></summary><code></code></details>
<details><summary><b><a href="https://www.ebi.ac.uk/">The European Bioinformatics Institute (EMBL-EBI)</b></a> | Database with bioinformatic information and tools<br></summary><code></code></details>
<details><summary><b><a href="https://biocyc.org">BioCyc</b></a> | Collection of pathway &amp; genomic databases for model organisms<br></summary><code></code></details>
<details><summary><b><a href="https://www.proteinatlas.org/">The Human Protein Atlas</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.fpbase.org/">FPbase</b></a> | $\color{#3BBB27}\textsf{\Large\kern{0.2cm}\normalsize [web]}$ | Fluorescent protein database<br></summary><code></code></details>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/sra">Sequence Read Archive (SRA)</b></a> | $\color{#3BBB27}\textsf{\Large\kern{0.2cm}\normalsize [web]}$ | Largest repository of high through sequencing data<br></summary><code></code></details>
<details><summary><b><a href="https://www.ebi.ac.uk/intact/home">IntAct</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.genecards.org/">GeneCards</b></a><br></summary><code></code></details>
</ul><p>Structural</p><ul>
<details><summary><b><a href="https://www.rcsb.org/">RCSB Protein Data Bank (PDB)</b></a> | Protein database with 3D structural information<br></summary><code></code></details>
</ul><p>RNA</p><ul>
<details><summary><b><a href="https://rnacentral.org/">RNAcentral</b></a> | The non-coding RNA sequence database<br></summary><code></code></details>
<details><summary><b><a href="https://www.mirbase.org/">miRBase</b></a> | $\color{#3BBB27}\textsf{\Large\kern{0.2cm}\normalsize [web]}$ | Largest updated microRNA database <br></summary><code></code></details>
<details><summary><b><a href="https://mirgenedb.org/">MirGeneDB</b></a> | $\color{#3BBB27}\textsf{\Large\kern{0.2cm}\normalsize [web]}$ | Well-annotated microRNA database<br></summary><code></code></details>
<details><summary><b><a href="https://dianalab.e-ce.uth.gr/mited/#/"">miTED</b></a> | $\color{#3BBB27}\textsf{\Large\kern{0.2cm}\normalsize [web]}$ | MicroRNA Tissue Expression Database<br></summary><code></code></details>
</ul><p>Aging</p><ul>
<details><summary><b><a href="https://circaage.shinyapps.io/circaage/">Aging Circadian Database</b></a> | Contains circadian gene expression data data from various ages and tissues<br></summary><code></code></details>
<details><summary><b><a href="https://ngdc.cncb.ac.cn/aging/index">Aging Atlas</b></a> | Collection of gene expression data related to the aging process<br></summary><code></code></details>
</ul><p>circRNAs</p><ul>
<details><summary><b><a href="http://www.circbase.org/">circBase</b></a> | Circular RNA database<br></summary><code></code></details>
<details><summary><b><a href="http://www.circbank.cn/index.html">circBank</b></a> | Circular RNA database<br></summary><code></code></details>
</ul><p>Species Specific</p><ul>
<details><summary><b><a href="https://www.yeastgenome.org/">SGD</b></a> | Comprehensive database on <em>Saccharomyces cerevisiae</em><br></summary><code></code></details>
<details><summary><b><a href="https://www.arabidopsis.org/">TAIR</b></a> | Contains data relevant to <em>Arabidopsis thaliana</em><br></summary><code></code></details>
<details><summary><b><a href="https://ecocyc.org/">EcoCyc</b></a> | Curated database on genome, transcriptional regulation transporters &amp; metabolic pathways in <em>Escherichia coli</em> K-12 MG1655<br></summary><code></code></details>
<details><summary><b><a href="https://regulondb.ccg.unam.mx/">RegulonDB Database</b></a> | Curated database on transcriptional regulation in <em>Escherichia coli</em> K-12<br></summary><code></code></details>
</ul><p>Miscellaneous</p><ul>
<details><summary><b><a href="https://gold.jgi.doe.gov/">JGI GOLD</b></a> | Collection of metagenomic and genomic data<br></summary><code></code></details>
<details><summary><b><a href="http://lowelab.ucsc.edu/GtRNAdb/">Genomic tRNA Database</b></a> | Database of automated annotated tRNA gene predictions using tRNAscan-SE<br></summary><code></code></details>
<details><summary><b><a href="https://regprecise.lbl.gov/index.jsp">Regprecise</b></a> | Contains curated and propagated predictions for regulons in prokaryotic genomes<br></summary><code></code></details>
<details><summary><b><a href="https://www.bioinformatics.org/molvis/atlas/atlas.htm">Atlas of Macromolecules</b></a> | Database containing data on about 160 macromolecules<br></summary><code></code></details>
<details><summary><b><a href="https://bionumbers.hms.harvard.edu/search.aspx">Bionumbers</b></a> | Describes useful number values in biology<br></summary><code></code></details>
<details><summary><b><a href="https://ngdc.cncb.ac.cn/databasecommons/">Database Commons</b></a> | Curated catalog of biological databases<br></summary><code></code></details>


</ul><h2 id="tool-platforms">Tool Platforms</h2>
<p>Webtools</p><ul>
<details><summary><b><a href="https://genomevolution.org/coge/">CoGe</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://toolkit.tuebingen.mpg.de/">MPI Bioinformatics Toolkit</b></a> <br></summary><code></code></details>
<details><summary><b><a href="http://www.softberry.com/">Softberry</b></a>  <br></summary><code></code></details>
<details><summary><b><a href="https://bio.tools/t">Elixir Bio.tools</b></a><br></summary><code></code></details>
<details><summary><b><a href="http://www.bioinformatics.org/sms2/index.html">Sequence Manipulation Suite (SMS)</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.bioinformatics.org/sewer/">SeWeR</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_server.html">Prabi</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://drmichaellevin.org/resources/software.html#BETSE">Levin Lab Tools</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://usegalaxy.org/">Galaxy</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.neb.com/en-us/tools-and-resources/interactive-tools">NEB Tools</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://nf-co.re/rnaseq/3.14.0">RNASeq Pipeline</b></a><br></summary><code></code></details>
</ul><p>Lab Specific Tool Suites</p><ul>
<details><summary><b><a href="https://bit.cs.tum.de/resources-data/resources/">GrimmLab</b></a><br></summary><code></code></details>
<details><summary><b><a href="http://quinlanlab.org/software.html">Quinlan Lab</b></a><br></summary><code></code></details>
</ul><p>Local Software</p><ul>
<details><summary><b><a href="http://ugene.net/">UGene</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.jalview.org/">Jalview</b></a><br></summary><code></code></details>

</ul><h2 id="sequence-alignment-tools">Sequence Alignment Tools</h2>
<p>Positions various sequences with similar portions in line with one another to observe possible homology</p><h3 id="two-sequence-alignment-tools">Two Sequence Alignment Tools</h3><ul>
<details><summary><b><a href="http://www.bioinformatics.org/sms2/pairwise_align_dna.html">Pairwise Align DNA</b></a><br></summary><code></code></details>
<details><summary><b><a href="http://www.bioinformatics.org/sms2/pairwise_align_protein.html">Pairwise Align Protein</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://proteininformationresource.org/pirwww/search/pairwise.shtml">PIR Pairwise Alignment</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.ebi.ac.uk/Tools/psa/lalign/">LALIGN</b></a><br></summary><code></code></details>

</ul><h3 id="multiple-sequence-alignment-tools">Multiple Sequence Alignment Tools</h3><ul>
<details><summary><b><a href="https://www.ebi.ac.uk/Tools/msa/clustalo/">Clustal Omega</b></a> <br></summary><code></code></details>
<details><summary><b><a href="http://multalin.toulouse.inra.fr/multalin/cgi-bin/multalin.pl">MultAlin</b></a> <br></summary><code></code></details>
<details><summary><b><a href="http://tcoffee.crg.cat/apps/tcoffee/result?rid=24cdc9c7">T-Coffee</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://www.ebi.ac.uk/Tools/msa/">EMBL-EBI MSA Tools</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.bioinformatics.org/msareveal/index2.html">MSAReveal</b></a><br></summary><code></code></details>

</ul><h2 id="dotplot-generators">Dotplot Generators</h2>
<p>Produces dot plots for comparing sequences</p><ul>
<details><summary><b><a href="https://dotlet.vital-it.ch/">Dotlet JS</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://myhits.sib.swiss/cgi-bin/dotlet">Dotlet</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://www.bioinformatics.nl/cgi-bin/emboss/dotmatcher">Dotmatcher</b></a><br></summary><code></code></details>

</ul><h3 id="genome-comparison">Genome Comparison</h3><ul>
<details><summary><b><a href="https://genomevolution.org/CoGe/SynMap.pl">CoGe: SynMap</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://genomevolution.org/coge/SynMap3D.pl">CoGe: SynMap3D</b></a> <br></summary><code></code></details>

</ul><h2 id="sequence-scanning">Sequence Scanning</h2>
<p>Specialized tools for scanning sequences against knowledgebase and specific functional areas of a sequence or genome </p><ul>
<details><summary><b><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</b></a> | The quintessential tool for identifying sequences<br></summary><code></code></details>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/orffinder/">NCBI: ORFfinder</b></a> | Searches for open reading frames in a sequence <br></summary><code></code></details>
<details><summary><b><a href="http://lowelab.ucsc.edu/tRNAscan-SE/">tRNAscan-SE </b></a> | Uses predictive software to scan for possible tRNAs<br></summary><code></code></details>
<details><summary><b><a href="http://exon.gatech.edu/GeneMark/">GeneMark gene prediction</b></a> | Implements unsupervised training to identify protein coding genes<br></summary><code></code></details>
<details><summary><b><a href="http://www.softberry.com/berry.phtml?topic=fgenesh&amp;group=programs&amp;subgroup=gfind">FGENESH</b></a> | Gene structure prediction tool<br></summary><code></code></details>
<details><summary><b><a href="https://github.com/ncbi/ribovore">Ribovore</b></a> | A collection of tools for subunit RNA detection<br></summary><code></code></details>
<details><summary><b><a href="https://www.bioinformatics.org/pecop/">PeCoP</b></a> | A PSI-BLAST tool for finding persistently conserved positions<br></summary><code></code></details>
<details><summary><b><a href="https://www.targetscan.org/vert_80/">TargetScanHuman 8.0</b></a> | Searches for predicted micro RNA targets in mammals<br></summary><code></code></details>
<details><summary><b><a href="https://www.cuilab.cn/sramp">SRAMP</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://github.com/Weeks-UNC/shapemapper2">ShapeMapper2</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://github.com/dincarnato/RNAFramework">RNAFramework</b></a><br></summary><code></code></details>

</ul><h2 id="sequence-visualization-editing">Sequence Visualization &amp; Editing</h2><ul>
<details><summary><b><a href="https://weblogo.berkeley.edu/">WebLogo About</b></a> | Generates sequence logos<br></summary><code></code></details>
<details><summary><b><a href="https://benchling.com/editor">Benchling</b></a> | Platform for conducting biotechnology research allowing for easy gene annotation, visualization, and more.<br></summary><code></code></details>
<details><summary><b><a href="https://jorgensen.biology.utah.edu/wayned/ape/">ApE (A Plasmid Editor)</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.bioinformatics.org/savvy/">Savvy</b></a> | Website for producing annotated plasmid maps<br></summary><code></code></details>
<details><summary><b><a href="https://old.nupack.org/">NUPACK</b></a> | Webserver for analyzing nucleic acid structures<br></summary><code></code></details>
<details><summary><b><a href="https://biomodel.uah.es/en/lab/cybertory/analysis/massager.htm">Sequence Massager</b></a> | Allows for easy sequence modification including automatic transcription<br></summary><code></code></details>
<details><summary><b><a href="https://web.expasy.org/translate/">Translate</b></a> | Easy tool for converting DNA/RNA sequences into amino acids<br></summary><code></code></details>

</ul><h2 id="motifs-protein-domains-identification">Motifs &amp; Protein Domains Identification</h2><ul>
<details><summary><b><a href="https://prosite.expasy.org/scanprosite/">ScanProsite</b></a> | Scans sequences against collection of motifs<br></summary><code></code></details>
<details><summary><b><a href="http://www.ebi.ac.uk/interpro/search/sequence/">InterPro</b></a> | Classifies proteins into families using predictive algorithm and has curated databases<br></summary><code></code></details>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi">NCBI Conserved Domain Search</b></a> | Identifies conserved domains within a sequence<br></summary><code></code></details>
<details><summary><b><a href="http://www.softberry.com/berry.phtml?topic=index&amp;group=programs&amp;subgroup=promoter">Softberry Tools</b></a> | Suite of tools for scanning for motifs and domains<br></summary><code></code></details>
<details><summary><b><a href="https://meme-suite.org/meme/index.html">Introduction MEME Suite</b></a> | A collection of motif-based analysis tools<br></summary><code></code></details>
</ul><p>Relevant Databases:</p><ul>
<details><summary><b><a href="https://scop.mrc-lmb.cam.ac.uk/">SCOP| Structural Classification of Proteins</b></a> | Contains detailed information concerning protein folds and related structures<br></summary><code></code></details>
<details><summary><b><a href="http://www.cathdb.info/">CATH: Protein Structure Classification Database at UCL</b></a> | Database of protein structures<br></summary><code></code></details>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml">Conserved Domains Database (CDD)</b></a> | Tools for analyzing conserved domains and protein classifications<br></summary><code></code></details>
<details><summary><b><a href="https://coxpresdb.jp/">COXPRESdb</b></a> | Database with information concerning co-regulated gene relationships<br></summary><code></code></details>

</ul><h2 id="structure-visualization">Structure Visualization</h2>
<p>Group of tools for producing 2D &amp; 3D structural data</p><ul>
<details><summary><b><a href="https://swissmodel.expasy.org/interactive">Swiss-Model</b></a> | 3D protein modeling with similar experimental and predicted models<br></summary><code></code></details>
<details><summary><b><a href="https://wlab.ethz.ch/protter/start/">Protter interactive protein feature visualization</b></a> | 2D protein visualizing tool of polypeptide sequence and predicted cellular location<br></summary><code></code></details>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/Structure/icn3d/full.html">iCn3D: Web-based 3D Structure Viewer</b></a> | Loads PDB files and generates 3D structures <br></summary><code></code></details>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/Structure/CN3D/cn3d.shtml">Cn3D</b></a> | 3D structure viewer<br></summary><code></code></details>
<details><summary><b><a href="https://zhanggroup.org//TM-align/">TM-align</b></a> | Algorithm for superpositioning two protein structures using residue-residue alignment<br></summary><code></code></details>
<details><summary><b><a href="https://zhanggroup.org/US-align/">US-align</b></a> | Extended from TM-align and compares 3D structures (proteins, RNA, &amp; DNA) <br></summary><code></code></details>
<details><summary><b><a href="https://zhanggroup.org/I-TASSER/">I-TASSER</b></a> | Agorithm to predict protein structures using a hierarchical approach<br></summary><code></code></details>
<details><summary><b><a href="https://rna.urmc.rochester.edu/RNAstructureWeb/">RNAstructure</b></a> | Contains tools for RNA secondary structure prediction<br></summary><code></code></details>
<details><summary><b><a href="https://server.poissonboltzmann.org/pdb2pqr">PDB2PQR</b></a> | Visualize electrostatic potential of a 3D protein<br></summary><code></code></details>
<details><summary><b><a href="http://www.sbg.bio.ic.ac.uk/~phyre2/html/page.cgi?id=index">Phyre2</b></a> | Simple modeling software that allows for one-to-one threading<br></summary><code></code></details>
<details><summary><b><a href="http://www.openrasmol.org/">Rasmol</b></a> | Program for molecular model visualization<br></summary><code></code></details>
<details><summary><b><a href="https://spdbv.unil.ch/">Swiss-PDB Viewer</b></a> | Viewing software for analyzising several protein structures at once<br></summary><code></code></details>
<details><summary><b><a href="http://honig.c2b2.columbia.edu/grasp">GRASP</b></a> | Macromolecule visualization tool<br></summary><code></code></details>
<details><summary><b><a href="http://www.cgl.ucsf.edu/chimera/">UCSF Chimera</b></a> | Program for visualization of molecular structures<br></summary><code></code></details>
<details><summary><b><a href="https://pymol.org/2/">Pymol</b></a> | Software for displaying and modifying macromolecules<br></summary><code></code></details>
<details><summary><b><a href="https://jmol.sourceforge.net/">Jmol</b></a> | Viewer for 3D chemical structures<br></summary><code></code></details>
<details><summary><b><a href="http://www.compbio.dundee.ac.uk/jpred4/index.html">Jpred4</b></a> | Protein secondary structure predictor<br></summary><code></code></details>
<details><summary><b><a href="https://molbiol-tools.ca/Protein_secondary_structure.htm">Protein Secondary Structure Tools</b></a> | Group of programs for predicting protein secondary structure<br></summary><code></code></details>
<details><summary><b><a href="https://www.rcsb.org/docs/additional-resources/molecular-graphics-software">Molecular Graphics Software</b></a> | List of resources for visualizing macromolecules and interactions<br></summary><code></code></details>
<details><summary><b><a href="https://ccsb.scripps.edu/projects/visualization/">MGLtools</b></a> | Visualization tool for observing macromolecule structure<br></summary><code></code></details>
<details><summary><b><a href="https://github.com/dincarnato/RNAFramework">RNAFramework</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.tbi.univie.ac.at/RNA/">ViennaRNA</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.samson-connect.net/">Samson</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://rna.urmc.rochester.edu/RNAstructure.html">RNAStructure</b></a><br></summary><code></code></details>
</ul><p>Relevant Databases:</p><ul>
<details><summary><b><a href="https://alphafold.ebi.ac.uk/">Alphafold</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://swissmodel.expasy.org/">Swiss-Model</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.rcsb.org/">RCSB PDB</b></a><br></summary><code></code></details>

</ul><h2 id="primer-development">Primer Development</h2><ul>
<details><summary><b><a href="https://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi">Primer3Plus</b></a> | Uses Primer3 software to produces primers to detect a given sequence<br></summary><code></code></details>
<details><summary><b><a href="https://bioinfo.ut.ee/primer3/">Primer3</b></a> | Original software for generating primers for sequences<br></summary><code></code></details>
<details><summary><b><a href="https://www.bioinformatics.org/primerx/">PrimerX</b></a> | Automates design of mutagenic PCR primers<br></summary><code></code></details>

</ul><h2 id="restriction-enzymatic-tools">Restriction Enzymatic Tools</h2><ul>
<details><summary><b><a href="https://www.neb.com/tools-and-resources/selection-charts/alphabetized-list-of-recognition-specificities">NEB List of Recognition Specificities</b></a> | Endonuclease recognition sequences and associated enzyme<br></summary><code></code></details>
<details><summary><b><a href="http://nc2.neb.com/NEBcutter2/">NEBcutter V2.0</b></a> | Identify cut sites for a DNA sequence<br></summary><code></code></details>

</ul><h2 id="genome-browsers">Genome Browsers</h2><ul>
<details><summary><b><a href="https://pubseed.theseed.org/?page=OrganismSelect">Seed Viewer</b></a> | Curated genomic data for comparative genomics<br></summary><code></code></details>
<details><summary><b><a href="https://www.pathogenomics.sfu.ca/islandviewer/">IslandViewer 4</b></a> | Tool for predicting genomic islands<br></summary><code></code></details>
<details><summary><b><a href="https://genome.ucsc.edu/index.html">UCSC Genome Browser</b></a> | General genome browser<br></summary><code></code></details>
<details><summary><b><a href="http://useast.ensembl.org/index.html">Ensembl</b></a> | Vertebrate genome browser<br></summary><code></code></details>
<details><summary><b><a href="http://bacteria.ensembl.org/index.html">Ensembl Bacteria</b></a> | Bacterial genomic data<br></summary><code></code></details>
<details><summary><b><a href="http://archaea.ucsc.edu/">UCSC Archaeal Genome Browser</b></a> | Archaeal genomic data <br></summary><code></code></details>
<details><summary><b><a href="https://www.pseudomonas.com/">Pseudomonas Genome DB</b></a> | Browses pseudomonas genome<br></summary><code></code></details>
<details><summary><b><a href="https://mage.genoscope.cns.fr/microscope/home/index.php">MicroScope: MAGE</b></a> | Microbial Genome Annotation &amp; Analysis Platform<br></summary><code></code></details>

</ul><h2 id="phylogeny-tools">Phylogeny Tools</h2>
<p>Generates phylogenetic trees</p><ul>
<details><summary><b><a href="https://ngphylogeny.fr/">NGPhylogeny.fr</b></a></p><ul><li><a href="http://phylogeny.lirmm.fr/phylo_cgi/index.cgi">Phylogeny.fr</b></a> | Previous version</li></ul></li>
<details><summary><b><a href="https://www.ebi.ac.uk/Tools/phylogeny/simple_phylogeny/">Simple Phylogeny</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://doua.prabi.fr/software/seaview">SeaView</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.megasoftware.net/">MEGA 11</b></a><br></summary><code></code></details>

</ul><h2 id="protein-analysis-tools">Protein Analysis Tools</h2><ul>
<details><summary><b><a href="https://phobius.sbc.su.se/">Phobius</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://services.healthtech.dtu.dk/service.php?SignalP-5.0">SignalP 5.0 Services DTU Health Tech</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://services.healthtech.dtu.dk/service.php?TMHMM-2.0">TMHMM 2.0 Services DTU Health Tech</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://services.healthtech.dtu.dk/service.php?TargetP-2.0">TargetP 2.0 Services DTU Health Tech</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://web.expasy.org/compute_pi/">ExPASy Compute pI/Mw tool</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://www.mechanobio.info/the-cell/protein-localization/">Protein Localization | MBInfo</b></a> <br></summary><code></code></details>
<details><summary><b><a href="http://imed.med.ucm.es/Tools/sias.html">Immunomedicine Group: Tools &gt;&gt; SIAS</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://mitominer.mrc-mbu.cam.ac.uk/release-4.0/begin.do">MitoMiner 4.0: Home</b></a> <br></summary><code></code></details>
<details><summary><b><a href="https://web.expasy.org/pratt/">PRATT</b></a> <br></summary><code></code></details>

</ul><h2 id="metagenomic-tools">Metagenomic Tools</h2>
<p>Tools and packages for conducting metagenomic analysis</p><ul>
<details><summary><b><a href="https://www.microbiomeanalyst.ca/">MicrobiomeAnalyst</b></a> | Statistical software for analyzing metagenomic data<br></summary><code></code></details>
<details><summary><b><a href="https://benjjneb.github.io/dada2/">DADA2</b></a> | Denoising package for eliminating spurious reads<br></summary><code></code></details>
<details><summary><b><a href="https://mothur.org/">Mothur</b></a> | Denoising package for filtering false reads<br></summary><code></code></details>
<details><summary><b><a href="https://qiime2.org/">Qiime2</b></a> | Suite of tools for metagenomic data processing and analysis<br></summary><code></code></details>

</ul><h2 id="conversion-software">Conversion Software</h2>
<p>Useful tools for converting different between different filetypes</p><ul>
<details><summary><b><a href="https://rnacomposer.cs.put.poznan.pl/tools;jsessionid=945AA52BC80AD4C9A009D898D83F51CB">RNAComposer</b></a> | Converts RNA structure files between different formats<br></summary><code></code></details>
<details><summary><b><a href="https://www.ebi.ac.uk/Tools/sfc/emboss_seqret/">EMBOSS Seqret</b></a> | Tool for reformatting sequence filetypes<br></summary><code></code></details>

</ul><h2 id="forums">Forums</h2>
<p>Helpful Forums for bioinformatics discussion</p><ul>
<details><summary><b><a href="https://www.biostars.org/t/Forum/">Biostars</b></a> | General bioinformatics forum<br></summary><code></code></details>
<details><summary><b><a href="https://www.seqanswers.com/forum/bioinformatics/bioinformatics-aa">SEQanswers</b></a> | Sequencing centric forum <br></summary><code></code></details>
<details><summary><b><a href="http://www.bioinformatics.org/forum/forum.php?forum_id=1300&amp;et=0">Bioinformatics.org</b></a> | General bioinformatics forum<br></summary><code></code></details>
<details><summary><b><a href="https://stackoverflow.com/">Stack Overflow</b></a> | General programming forum<br></summary><code></code></details>
<details><summary><b><a href="https://omicsforum.ca/">OmicsForum</b></a> | Community for various analysis tools related to metabolomics, metagenomics, proteomics, and more<br></summary><code></code></details>
<details><summary><b><a href="https://forum.qiime2.org/">QIIME2 Forum</b></a> | Forum for the QIIME2 platform<br></summary><code></code></details>

</ul><h2 id="old-decommissioned-databases-tools">Old/Decommissioned Databases &amp; Tools</h2><ul>
<details><summary><b><a href="http://www.microbesonline.org/">MicrobesOnline</b></a> | Database centered on microbial data CAUTION Last updated is June 2009<br></summary><code></code></details>
<details><summary><b><a href="http://xfam.org/">Xfam</b></a> | Decommissioned collection of databases for families of Proteins, DNA, and RNA CAUTION Inactive<br></summary><code></code></details>
<details><summary><b><a href="http://www.cbs.dtu.dk/services/RNAmmer/">RNAmmer 1.2 Server</b></a> | Identified related sequences to ribosomal subunits<br></summary><code></code></details>
<details><summary><b><a href="http://www.genebee.msu.su/services/dhm/advanced.html">GeneBee Motifs</b></a> | Program for identifying motifs within a sequence<br></summary><code></code></details>

</ul><h2 id="forums">Docs</h2>
</ul><p>General</p><ul>
<details><summary><b><a href="https://www.python.org/doc/">Python</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://daler.github.io/pybedtools/index.html">PyBedtools</b></a><br></summary><code></code></details>
</ul><p>SLURM</p><ul>
<details><summary><b><a href="https://help.rc.ufl.edu/doc/UFRC_Help_and_Documentation">UF Research Computing Documentation</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://slurm.schedmd.com/documentation.html">Official SLURM Documentation</b></a><br></summary><code></code></details>
</ul><p>BenchtopCloning Guides</p><ul>
<details><summary><b><a href="https://blog.addgene.org/plasmids-101-cre-lox">addgene Cre-lox</b></a><br></summary><code></code></details>

</ul><h2 id="forums">Commandline Tools</h2>
</ul><p>General</p><ul>
<details><summary><b><a href="https://bedtools.readthedocs.io/en/latest/index.html">bedtools</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://www.htslib.org/">samtools</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://regtools.readthedocs.io/en/latest/">regtools</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://rseqc.sourceforge.net/">RSeQC</b></a> | An RNA-seq Quality Control Package<br></summary><code></code></details>
</ul><p>Docs</p><ul>
<details><summary><b><a href="https://samtools.github.io/hts-specs/SAMv1.pdf">Samtools</b></a><br></summary><code></code></details>
</ul><p>Uncategorized</p><ul>
<details><summary><b><a href="https://www.ncbi.nlm.nih.gov/geo/">Gene Expression Omnibus (GEO)</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://maayanlab.cloud/Harmonizome/">Harmonizome</b></a><br></summary><code></code></details>
<details><summary><b><a href="https://maayanlab.cloud/Enrichr/">Enrichr</b></a><br></summary><code></code></details>

https://guides.lib.berkeley.edu/bioinformatics"

https://en.wikipedia.org/wiki/List_of_open-source_bioinformatics_software

https://github.com/pysam-developers/pysam

https://rseqc.sourceforge.net/

https://rnaseq-mats.sourceforge.io/ Multivariate Analysis of Transcript Splicing (MATS) https://github.com/Xinglab/rmats-turbo/tree/v4.2.0

http://www.usadellab.org/cms/?page=trimmomatic

https://cutadapt.readthedocs.io/en/stable/

https://www.bioinformatics.babraham.ac.uk/projects/fastqc/  - https://mugenomicscore.missouri.edu/PDF/FastQC_Manual.pdf

https://ccb.jhu.edu/software/stringtie/gff.shtml#format - gffread

https://jgi.doe.gov/data-and-tools/software-tools/bbtools/

https://github.com/ncbi/sra-tools

https://github.com/OpenGene/fastp

https://github.com/Hengfu-Yin/IsoSplitter

https://github.com/griffithlab/regtools

https://github.com/TimoLassmann/samstat

https://github.com/sortmerna/sortmerna

https://mafft.cbrc.jp/alignment/software/b

https://daehwankimlab.github.io/hisat2/

https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#the-bowtie2-aligner
http://qualimap.conesalab.org/doc_html/index.html#

https://app.fluorofinder.com/svs_portal

RNA-seq
https://github.com/alexdobin/STAR

https://github.com/rajewsky-lab/mirdeep2

https://www.thermofisher.com/order/fluorescence-spectraviewer/#!/   https://www.thermofisher.com/us/en/home/life-science/cell-analysis/fluorophores.html

https://www.ncbi.nlm.nih.gov/bioproject Bioproject section??!?!?

http://rna.tbi.univie.ac.at//cgi-bin/RNAWebSuite/RNAfold.cgi

http://contra.stanford.edu/contrafold/

https://cryosparc.com/

https://cistem.org/
